File: Python/SQL/SQL.py
def select(t, cl, ext=''):
	return 'SELECT '+','.join(cl)+' FROM '+t+ext

def insert(t, vl, cl = None):
	s = 'INSERT INTO '+t
	if(not cl is None):
		s += '('+','.join(cl)+')'
	v = map(lambda e: '"'+e+'"' if (type(e) is str) else str(e), vl)
	return s+' VALUES('+','.join(v)+')'

File: JavaScript/_server/db/SQL.js
module.exports = {
	insert: function(t, vl, nm = null){
		let s = 'INSERT INTO '+t;
		if(nm){
			s += '('+nm.join(',')+')';
		}
		return s+' VALUES('+vl.map(e=>typeof e!='string'?e:'"'+e+'"').join(',')+')';
	},
	select: function(t, nm, ext = ''){
		return 'SELECT '+nm.join(',')+' FROM '+t+ext;
	},
	delete: function(t, ext){
		return 'DELETE FROM '+t+' WHERE '+ext;
	}
}
